pkgs/os-specific/darwin/iproute2mac/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, python }:
+
+stdenv.mkDerivation rec {
+  version = "1.2.1";
+  name = "iproute2mac-${version}";
+
+  src = fetchFromGitHub {
+    owner = "brona";
+    repo = "iproute2mac";
+    rev = "v${version}";
+    sha256 = "1n6la7blbxza2m79cpnywsavhzsdv4gzdxrkly4dppyidjg6jy1h";
+  };
+
+  buildInputs = [ python ];
+
+  __impureHostDeps = [
```

Comment:
This is only necessary if you're actually calling these in the builder. You should be able to leave this out.

Reply:
ok, wasn't aware. will strip them out.

Reply:
Yeah, and I'm not even sure if `/usr/bin` is allowed.

Reply:
@LnL7 from the builder? Don't know too, I just thought I should probably document impure usages somewhere. PR updated with `__impureHostDeps` removed.

Reply:
For reference, these are the only prefixes that this works for unless this is overridden in nix.conf with `allowed-impure-host-deps`. 
https://github.com/NixOS/nix/blob/e7cb2847ab1cec48eac6a86c56885b3f0df76275/src/libstore/globals.cc#L22

This is used to open up the build sandbox when sandboxing is enabled, allowing builds to open up arbitrary paths would be a security concern.

Reply:
thanks for the explanation!